rust/worker/src/blockstore/arrow/config.rs

```diff
@@ -13,4 +15,16 @@ pub(crate) struct ArrowBlockfileProviderConfig {
     // but the only configuration that is needed is the max_block_size_bytes
     // so for now we just hoid this configuration in the ArrowBlockfileProviderConfig.
     pub(crate) max_block_size_bytes: usize,
```

Comment:
should this be pushed into the block_manager_config since we introduced that now? And is this comment irrelevant and should it be removed?

Reply:
I can address this in a followup PR. I will create a task.